salt/config/__init__.py

```diff
@@ -3284,6 +3292,42 @@ def _cache_id(minion_id, cache_file):
         log.error('Could not cache minion ID: {0}'.format(exc))
 
 
+def eval_minion_id_func(opts):
+    '''
+    Evaluate minion ID function if 'minion_id_type' is 'func'
+    and return the result
+    '''
+    if '__minion_id_func_evaluated' in opts:
+        return opts['id']
+
+    import salt.loader as loader
```

Comment:
This was put here purposely to avoid this import in other cases. Is there a specific reason why it should be above in the imports, or this this just pedantic style enforcement?